src/helm/env.d/dev/values.desk.yaml.gotmpl

```diff
@@ -47,7 +47,8 @@ backend:
         name: backend
         key: OIDC_RP_CLIENT_SECRET
     OIDC_RP_SIGN_ALGO: RS256
-    OIDC_RP_SCOPES: "openid email siret"
+    OIDC_RP_SCOPES: "openid email siret given_name usual_name"
```

Comment:
@qbey I'm unsure whether I should put the same scope on the keycloak config or if it doesn't matter because keycloak doesn't use these scopes anyway.

Reply:
Keycloak uses the standard : openid email profile siret